pkg/monitortests/network/disruptionserviceloadbalancer/monitortest.go

```diff
@@ -344,3 +355,31 @@ func httpGetNoConnectionPoolTimeout(url string, timeout time.Duration) (*http.Re
 
 	return client.Get(url)
 }
+
+// checkHostnameReady debugs the master node in the PowerVS or IBMCloud cluster
+// and verifies the hostname is active before hitting through the LoadBalancer
+func checkHostnameReady(tcpService *corev1.Service, nodeTgt string) error {
+	for i := 0; i < 60; i++ {
+		lbTgt := tcpService.Status.LoadBalancer.Ingress[0].Hostname
+		cmd := exec.Command("oc", "debug", nodeTgt, "--", "/bin/bash", "-c", "dig +short "+lbTgt)
+		out := &bytes.Buffer{}
+		errOut := &bytes.Buffer{}
+		cmd.Stdout = out
+		cmd.Stderr = errOut
+
+		if err := cmd.Run(); err != nil {
```

Comment:
I'm not super familiar with the run command, but it looks like the expectation is for this to run synchronously. My suggestion would be to set an explicit timeout to cover a case where scheduling the debug pod gets stuck in limbo for whatever reason. If we have to cancel the command upon said timeout, it should be safe to fail out, since being unable to schedule for whatever reason is something of concern. (But there should probably an error message indicating what happened).

Reply:
Oh great - point, let me coordinate with Jason

Reply:
Addressed in latest commit